lib/travis/build/script/shared/rvm.rb

```diff
@@ -114,6 +115,10 @@ def write_default_gems
             sh.mkdir '$rvm_path/gemsets', recursive: true, echo: false
             sh.cmd 'echo -e "gem-wrappers\nrubygems-bundler\nbundler\nrake\nrvm\n" > $rvm_path/gemsets/global.gems', echo: false, timing: false
           end
+
+          def force_187_p371(version)
+            version.gsub(/^1\.8\.7.*$/,'1.8.7-p371')
```

Comment:
Shouldn't this be `sub` instead of `gsub`?  Also, nitpick: space after the comma plz